plugins/browser-plugin-form-tracking/src/helpers.ts

```diff
@@ -317,15 +298,17 @@ function getFormSubmissionListener(
     var elt = e.target as HTMLFormElement;
     var innerElements = getInnerFormElements(trackingMarker, elt);
     innerElements.forEach(function (innerElement) {
-      innerElement.value = config.fieldTransform(innerElement.value, innerElement) ?? innerElement.value;
+      var eltData = innerElement.elementData;
+      eltData.value = config.fieldTransform(eltData.value, eltData, innerElement.originalElement) ?? eltData.value;
```

Comment:
This has been the behaviour also before, but I am a bit confused why this part is different from the `change_form` event above? Here we coalesce `?? eltData.value` in case the `fieldTransform` returns a null value, but in in the change form we don't. Also, in the change form event, we exclude unchecked checkboxes but here we don't.

Am I missing something here? At least the coalesce behaviour should be the same – I think if the `fieldTransform` function returns `null`, we should keep it `null`. What do you think?

Reply:
Oh okay, that makes sense